overrides/scripts/createstages.zs

```diff
@@ -377,9 +377,7 @@ var itemsCursed = [
 
 
 //Eye gates
-	ItemStages.restrict(<item:dungeons_gear:satchel_of_elixirs>.anyDamage(), "witch_eye").preventInventory(false).preventPickup(false).setHiddenInJEI(false);
```

Comment:
This should be included in the dev build

Reply:
Well, all 3 lines thast seem to be removed

Reply:
isnt dungeons gear removed so this is fine?